recipes-kernel/linux/files/kernel-debug.config

```diff
@@ -0,0 +1,21 @@
+# Disable KASLR
+# CONFIG_RANDOMIZE_BASE is not set
+
+# Coverage collection.
+CONFIG_KCOV=y
+CONFIG_KCOV_ENABLE_COMPARISONS=y
+
+# Debug info for symbolization.
+CONFIG_DEBUG_INFO=y
+
+# Memory bug detector
+CONFIG_KASAN=y
+CONFIG_KASAN_INLINE=y
+
+# Required for Debian Stretch
```

Comment:
What does this mean?

Reply:
I copied these setting from [setup_ubuntu-host_qemu-vm_x86-64-kernel.md](https://github.com/google/syzkaller/blob/master/docs/linux/setup_ubuntu-host_qemu-vm_x86-64-kernel.md) but it doesn't need, so I removed them.
Also I update configs which followed [setup_linux-host_qemu-vm_arm64-kernel.md](https://github.com/google/syzkaller/blob/master/docs/linux/setup_linux-host_qemu-vm_arm64-kernel.md).